mne/decoding/time_gen.py

```diff
@@ -390,11 +446,13 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode,
     ----------
     X : ndarray, shape (n_epochs, n_features, n_times)
         To-be-fitted data.
-    estimators : array-like, shape (n_times, n_folds)
-        Array of scikit-learn classifiers fitted in cross-validation.
-    slices : list
-        List of slices selecting data from X from which is prediction is
-        generated.
+    estimators : list of array-like, shape (n_times, n_folds)
+        List of array of scikit-learn classifiers fitted in cross-validation.
```

Comment:
Generally speaking, I'm pretty pessimistic on a massive change to the GAT workflow. I don't think perfect alignment with sklearn API conventions should be a goal in the first place. This is MNE, we have Epochs objects for a reason; is there a *real* benefit to being able to pass arrays instead? GAT primarily doesn't give you classifications, it gives you a matrix of pattern similarities.
What should be the goal is being able to plug sklearn objects into GAT at the desired places (e.g., passing classifiers or pipelines as `clf`, which works as it should be right now).

If one wants to improve fit with sklearn conventions, as something that's more likely to be an actual improvement, I think it could be nice to make it easier to score stuff using sklearn metrics (e.g., no longer needing custom scorers for predict methods giving multidimensional predictions).